### PR TITLE
change margin of Election Alerts page

### PR DESCRIPTION
**Pull request by zjiao19**
## Overview 
35. Get Election Alerts form: Increase margins to 30px. 
36. Change link styles to bold and underline, in black.

## Test Plan 
Go to /election-reminders and inspect elements to make sure they align correctly.

### CODE REVIEW

**Comment by MazharulIslam-Naim** [comment]
Hey @zjiao19 thanks for working on this.

The left side seems to be correct but the right side is still a little to close and the top message seems to be overflowing its container. I saw that your adding margins to individual components instead you should remove the left and right padding and margins and just add the margin to the bigger container. 

Let me know if you need help.
image

**Comment by zjiao19** [comment]
Thank you! I fixed the right side margin and move the margin to the bigger container. Please let me know if there's anything else.

File: src/components/ElectionReminders/ElectionReminder.scss [hunk 1]
```diff
@@ -38,11 +39,17 @@
...
   .at-legend {
     font-family: "Lato", sans-serif !important;
-    padding-right: 18px !important;
+    width: auto;
   }
 
+
   .reminders-completed {
     font-family: "Lato", sans-serif;
     color: black;
```
Comment (MazharulIslam-Naim) [inline src/components/ElectionReminders/ElectionReminder.scss:55]: Can you the margin-left here to `margin: 0px 30px;`

**Review by MazharulIslam-Naim — CHANGES_REQUESTED** [review]

File: src/components/ElectionReminders/ElectionReminder.scss (hunk 1 above)
Comment (zjiao19) [reply src/components/ElectionReminders/ElectionReminder.scss:55]: Sure!

**Review by MazharulIslam-Naim — APPROVED** [review]
@zjiao19 Great work!

**Pull request merged**